### DISCUSSION TITLE
Change Task Manager Title

### DISCUSSION THREAD

**Discussion by Antkky** [tauri-apps/tauri#10332] [Q&A] [answered]
I was looking at my task manager to see how much of my computers resources my Tauri app is using and noticed that the name for it is "A Tauri App", This seems kind of unprofessional in the eyes of users but I can't for the life of me seem to be able to change it.

This is the only spot on my computer I can find that says "A Tauri App" instead of my desired "Apexa Journal"

![Screenshot 2024-07-20 034158](https://github.com/user-attachments/assets/e1a83350-37ee-4f5e-8793-5124ba12fa10)

here is my Tauri config File
![image](https://github.com/user-attachments/assets/fe41cc16-4915-4827-8974-63c49840fd48)

**Comment by HeyMyian** [comment, marked as answer]
You're looking for `shortDescription`, it goes as a child of "bundle" in your config file: https://tauri.app/v1/api/config/#bundleconfig.shortdescription
